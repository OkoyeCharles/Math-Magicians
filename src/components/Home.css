.home__container header {
  display: flex;
  flex-direction: column;
  gap: 1em;
  padding: 2em 1.6em;
}

.home__container header h1 {
  font-size: 2em;
  color: #444;
}

.home__container header h2 {
  position: relative;
  font-size: 1.5em;
  font-weight: 400;
}

.home__container header h2::after {
  z-index: -1;
  position: absolute;
  content: '';
  width: 100%;
  height: 100%;
  top: 0.5em;
  left: 0.5em;
  background: rgba(0, 0, 0, 0.074);
}

.home__container > div:nth-child(2) {
  background: #222;
  padding: 2em 1.6em;
  height: 50vh;
}

.home__container > div:nth-child(2) p {
  color: #eee;
  font-weight: 300;
  font-size: 1.25em;
}

@media screen and (min-width: 800px) {
  .home__container header {
    background-image: url(./images/calc.svg);
    background-position: 100% -50%;
    background-repeat: no-repeat;
    padding: 2em 4em;
  }

  .home__container header h1 {
    font-size: 3em;
  }

  .home__container > header h2 {
    font-size: 2em;
    width: fit-content;
  }

  .home__container > div:nth-child(2) {
    z-index: 1;
    padding: 2em 4em;
    height: 50vh;
  }

  .home__container > div:nth-child(2) p {
    width: 70%;
  }
}
