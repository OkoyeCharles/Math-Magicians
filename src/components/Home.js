import React from 'react';
import '../styles/Home.css';

function Home() {
  return (
    <div className="home__container">
      <header>
        <h1>Welcome to our Page!</h1>
        <h2>
          Making Calculations Accesible anywhere, anytime, anyplace.
        </h2>
      </header>

      <div>
        <p>
          If you ever have a hard time calculating complex equations or finding the right
          operator Math magicians is here for you just with the click of a button.
          <br />
          Here at math magicians we have made calculating equations accesible for anyone...
        </p>
      </div>
    </div>
  );
}

export default Home;
