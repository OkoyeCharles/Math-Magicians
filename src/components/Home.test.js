import React from 'react';
import renderer from 'react-test-renderer';
import Home from './Home';

describe('correctlly renders a component', () => {
  test('should render Home component', () => {
    const tree = renderer.create(<Home />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});