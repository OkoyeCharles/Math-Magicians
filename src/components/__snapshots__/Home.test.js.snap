// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`correctlly renders a component should render Home component 1`] = `
<div
  className="home__container"
>
  <header>
    <h1>
      Welcome to our Page!
    </h1>
    <h2>
      Making Calculations Accesible anywhere, anytime, anyplace.
    </h2>
  </header>
  <div>
    <p>
      If you ever have a hard time calculating complex equations or finding the right operator Math magicians is here for you just with the click of a button.
      <br />
      Here at math magicians we have made calculating equations accesible for anyone...
    </p>
  </div>
</div>
`;
